### PR TITLE
feat: :sparkles: added search to home page

### PR DESCRIPTION
**Pull request by snenenenenenene**

### CODE REVIEW

File: app/controllers/home.ts [hunk 1]
```diff
@@ -28,11 +28,26 @@ export default class HomeController extends Controller {
...
+  }
+
+  @action
+  handleChange(event: Event) {
+    this.value = (event.target as HTMLInputElement).value;
+  }
 
   @action handleMunicipalitySelect() {
     this.loading = true;
+    console.log(this.value);
```
Comment (aliokan) [inline app/controllers/home.ts:47]: **Issue:** @snenenenenenene oups ;)

File: app/controllers/home.ts [hunk 2]
```diff
@@ -28,11 +28,26 @@ export default class HomeController extends Controller {
   ) {
     this.selectedMunicipalities = selectedMunicipalities;
   }
+  /** Handles keyword search on homepage */
+  @tracked value = '';
```
Comment (aliokan) [inline app/controllers/home.ts:32]: **Nitpick:** Using a more meaningful name would improve readability and avoid confusion if other input text is added in the future 🙃

**Review by aliokan — CHANGES_REQUESTED** [review]
@snenenenenenene I first thought that it was worth creating a component to encapsulate the input logic. But after all, there is a good chance that the input keyword on home and list page will have different behavior or design in the future. 

Except the remaining `console.log` we are good to move forward. 

Good job! :+1:

**Comment by snenenenenenene** [comment]
@aliokan, fixed it! Thanks for the feedback <3

**Review by aliokan — APPROVED** [review]
Tested locally, looks good! :+1: 

LGTM! :rocket:

**Pull request merged**